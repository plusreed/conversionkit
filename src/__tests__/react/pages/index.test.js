import React from 'react';
import renderer from 'react-test-renderer'
import Index from '../../../app/pages';

it('matches snapshot', () => {
  const tree = renderer
    .create(<Index />)
    .toJSON()

  expect(tree).toMatchSnapshot()
});
