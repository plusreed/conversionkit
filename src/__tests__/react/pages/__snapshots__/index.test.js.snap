// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="p-2 mx-auto text-center bg-gray-900 w-auto"
>
  <h1
    className="font-bold text-xl text-gray-100"
  >
    Waiting for ConversionCore interfaces...
  </h1>
</div>
`;
